cluster/examples/kubernetes/ceph/cluster-minimal.yaml

```diff
@@ -16,7 +16,7 @@ metadata:
   namespace: rook-ceph
 spec:
   cephVersion:
-    image: ceph/ceph:v14.2.1-20190430
+    image: ceph/ceph:latest-nautlius-devel
```

Comment:
the repo name and the tag name are incorrect.

It's ceph/daemon-base:latest-nautilus-devel